core/src/main/java/org/commonjava/indy/core/content/DefaultContentManager.java

```diff
@@ -311,7 +311,16 @@ private boolean checkListingMask( final ArtifactStore store, final String path )
 
         AbstractRepository repo = (AbstractRepository) store;
         Set<String> maskPatterns = repo.getPathMaskPatterns();
-        logger.debug( "Checking mask in: {}, type: {}, patterns: {}", repo.getName(), repo.getKey().getType(), maskPatterns );
+
+        if ( logger.isTraceEnabled() )
+        {
+            logger.trace( "Checking mask in: {}, type: {}, patterns: {}", repo.getName(), repo.getKey().getType(),
+                          maskPatterns );
+        }
+        else
+        {
+            logger.debug( "Checking mask in: {}, type: {}", repo.getName(), repo.getKey().getType() );
+        }
```

Comment:
@ruhan1 The problem is that the PR #725 is fixing this API incompatible, but seems it has test failure on CI. Not sure what happened on that now. I think @yma88 is still checking on that.